src/flownet/realization/_schedule.py

```diff
@@ -315,7 +315,7 @@ def get_first_date(self) -> datetime.date:
 
     def get_keywords(
         self,
-        date: datetime.date = None,
+        date: Optional[Union[List[datetime.date], datetime.date]] = None,
```

Comment:
Maybe change to "dates" to reflect it is not just only 1?
```suggestion
        dates: Optional[Union[List[datetime.date], datetime.date]] = None,
```